Workbooks/Log Analytics - Network Dependencies/categoryResources.json

```diff
@@ -0,0 +1,3 @@
+{
+    "en-us": {"name":"Network Dependencies", "description": "Look for network dependencies of your Virtual Machines.", "order": 200}
+}
```

Comment:
Would you add an empty newline at the end of every JSON file?

Reply:
Thanks.